### ISSUE TITLE
Loading state not announced to screen readers

### ISSUE THREAD

**Issue by geoffrich** [AlaskaAirlines/auro-loader#16]
**Do some research on what current solutions are out there and better understand the issue.
Work with ux what the correct experience should be.  It's possible that we don't want to say anything.**

## Describe the bug
The loading state of the button is not conveyed to screen readers.

## To Reproduce
See [this CodePen](https://codepen.io/geoff-rich/pen/LYxarGp).

Tab to the button and trigger it. It will enter a loading state. 

When the loading state is triggered, NVDA reads out "unavailable", presumably because the button is disabled. This does not properly convey a loading state.

## Expected behavior
The message read by the screen reader includes the text "loading".

## Desktop (please complete the following information):
 - OS: Windows 10
 - Browser: Firefox w/ NVDA

## Additional context
This likely needs to be fixed in auro-loader, but I'm creating the issue here because it directly impacts the button.

## Bug exit criteria 

This issue will be considered complete once a user can click on an element in the browser and the screen reader will announce that the button has moved to an active state per the user's interaction.

**Comment by blackfalcon** [comment]
This issue needs a little more research as to what is considered best practices with loaders and AT SRs feedback to users. 

@blackfalcon will reach out to UX research team for any direction they may have.
